apps/server/src/modules/board/uc/board.uc.ts

```diff
@@ -181,7 +181,7 @@ export class BoardUc {
 
 			return { id: room.schoolId, type: StorageLocation.SCHOOL };
 		}
-
-		throw new Error(`Cannot get storage location reference for context type ${context.type as string}`);
+		/* istanbul ignore next */
```

Comment:
we wana keep that?

Reply:
The "istanbul ignore" is because that scenario is difficult to test because we would need to set a context type that is not in the `BoardExternalReferenceType` enum. So the error is just for the very unlikely case that we get such a context type. And the `throw` is needed because the function wouldn't return anything otherwise.